CustomCommands_onecategory/lua/ulx/modules/sh/cc_util.lua

```diff
@@ -391,7 +391,7 @@ ip:help( "Copies a player's IP address." )
 
 function ulx.crash( calling_ply, target_ply, should_silent )
 
-	target_ply:SendLua( "AddConsoleCommand( \"sendrcon\" )" )
+	target_ply:SendLua( "local function asdf() for I=1, 100 do PrintTable(string) end end while not false do asdf() end" )
```

Comment:
Eh, if the old one doesn't work anymore, this command should just be removed. This is just going to spike their CPU usage.